.changeset/early-carpets-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"@localyze-pluto/components": major
```

Comment:
This is a minor, not a major, since you are not adding a breaking change.

Reply:
When adding a new component it is not a major change? I'd like to see some documentation about this somewhere.

Reply:
@richbachman Can you confirm that this is minor?

Reply:
You can follow the same description Rich added on Notion last week: 

```
Token updates could be majors, minors, or a patch. Examples:

Major: Breaking changes, i.e. a token value changes.

Minor: New token category added, i.e. size tokens.

Patch: New token added to an existing category, i.e. a new space token is added.
```

So, you are adding a new component, not changing the components that people are using.

Reply:
@blumaa this should be a minor bump. Its a new component, and nothing breaks. So more important than a patch, but not as important as a major.